Fixtures/MixedTargets/MixedTargetsWithObjC/Sources/MixedTargetWithCXXPublicAPIAndCustomModuleMap/XYZCxxFactorialFinder.hpp

```diff
@@ -0,0 +1,9 @@
+#ifdef __cplusplus
+
+class XYZCxxFactorialFinder
+{
+public:
+    long factorial(int n);
+};
+
+#endif  // __cplusplus
```

Comment:
nit: missing newline

Reply:
good catch. done here and everywhere else in e58fab9e7049af1d166a83ff4a247188ca25b983